gitlab/README.md

```diff
@@ -0,0 +1,28 @@
+# Gitea
+
+[Gitlab](https://gitlab.com/) is a git repository host, similar to GitHub, but
+entirely self-hosted. More description forthcoming...
+
+## Configuration
+
+Copy `.env-dist` to `.env`, and edit variables accordingly. 
+
+ * `GITLAB_TRAEFIK_HOST` to the external domain name forwarded from traefik, eg.
+   `git.example.com`
+
+Config documention forthcoming...
+
+## Initial setup
+
+Bring up the service with `docker-compose up -d`
```

Comment:
Lets use the `make` style to describe how to bringing up the service.

make config
make install
make open